queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT date_of_birth FROM animals WHERE date_of_birth BETWEEN '2016/01/01' AND '2019/12/31';
```

Comment:
- [ ]  This query returns the `date_of_birth` of all animals that  born between 2016 and 2019. However, we are asked to list the `name` of all animals born between 2016 and 2019. A good query would be `SELECT name FROM animals WHERE date_of_birth BETWEEN '2016/01/01' AND '2019/12/31';`